authors/settings.py

```diff
@@ -197,13 +197,17 @@
 # called `INSTALLED_APPS`.
 AUTH_USER_MODEL = 'authentication.User'
 
+
 REST_FRAMEWORK = {
     'EXCEPTION_HANDLER': 'authors.apps.core.exceptions.core_exception_handler',
     'NON_FIELD_ERRORS_KEY': 'error',
 
     'DEFAULT_AUTHENTICATION_CLASSES': (
         'authors.apps.authentication.backends.JWTAuthentication',
     ),
+
+    'DEFAULT_PAGINATION_CLASS': 'rest_framework.pagination.LimitOffsetPagination',
+    'PAGE_SIZE': 4
```

Comment:
4 items per page are too few. Raise to 8

Reply:
The limit indicates the maximum number of items to return and is equivalent to the page_size in other styles. This can be changed to the count one wants when making a query. 
example: 
`http://localhost:8000/api/articles/?limit=8&offset=12`